llvm/lib/Analysis/InlineAdvisor.cpp

```diff
@@ -206,6 +214,20 @@ void InlineAdvice::recordInliningWithCalleeDeleted() {
 AnalysisKey InlineAdvisorAnalysis::Key;
 AnalysisKey PluginInlineAdvisorAnalysis::Key;
 
+bool InlineAdvisorAnalysis::initializeIR2VecVocab(Module &M,
```

Comment:
nit: `initializeIR2VecVocabIfRequested`

because you're OK with it not being requested and then there's no init-ing. Makes reading the call site easier